packages/lodestar/src/network/gossip/gossipsub.ts

```diff
@@ -1,24 +1,24 @@
-import PeerId from "peer-id";
 import Gossipsub from "libp2p-gossipsub";
```

Comment:
fyi https://github.com/ethereum/eth2.0-specs/pull/2044
it isn't in a release yet, but we'll need to support it soon enough.
This might require an update to gossipsub, not sure. Currently, topics are `string`, and I think there may be issues with this one way or another.

Reply:
I only see the way to calculate message-id there which is not a big change, are there other PRs about topics? Anyway it's good to notice that, we should focus on libp2p migration to make sure gossip works fine with medalla first.
In the mean time, topics are still string in dev branch.